2021-05-19 22:58:06,046 INFO    MainThread:709772 [wandb_setup.py:_flush():69] setting env: {}
2021-05-19 22:58:06,046 INFO    MainThread:709772 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-19 22:58:06,046 INFO    MainThread:709772 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210519_225806-cqulxkfg/logs/debug.log
2021-05-19 22:58:06,046 INFO    MainThread:709772 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/Classification/Heart-Disease-UCI/wandb/run-20210519_225806-cqulxkfg/logs/debug-internal.log
2021-05-19 22:58:06,047 INFO    MainThread:709772 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f0c2477a828>
2021-05-19 22:58:06,047 INFO    MainThread:709772 [wandb_init.py:init():369] calling init triggers
2021-05-19 22:58:06,047 INFO    MainThread:709772 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-19 22:58:06,047 INFO    MainThread:709772 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 7sndhaf3
2021-05-19 22:58:06,049 INFO    MainThread:709772 [wandb_run.py:finish():1208] finishing run ranuga-d/Heart-Disease-UCI/7sndhaf3
2021-05-19 22:58:06,049 INFO    MainThread:709772 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-19 22:58:06,049 INFO    MainThread:709772 [wandb_run.py:_restore():1507] restore
2021-05-19 22:58:06,051 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 721
  total_bytes: 721
}

2021-05-19 22:58:06,618 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 721
  total_bytes: 721
}

2021-05-19 22:58:06,776 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 721
  total_bytes: 17521
}

2021-05-19 22:58:07,564 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:07,667 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:07,768 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:07,870 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:07,971 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:08,073 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:08,175 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:08,276 INFO    MainThread:709772 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 17523
  total_bytes: 17523
}

2021-05-19 22:58:09,575 INFO    MainThread:709772 [wandb_run.py:_show_files():1879] logging synced files
2021-05-19 22:58:09,579 INFO    MainThread:709772 [wandb_init.py:init():418] starting backend
2021-05-19 22:58:09,579 INFO    MainThread:709772 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-19 22:58:09,581 INFO    MainThread:709772 [backend.py:ensure_launched():132] starting backend process...
2021-05-19 22:58:09,614 INFO    MainThread:709772 [backend.py:ensure_launched():137] started backend process with pid: 712441
2021-05-19 22:58:09,615 INFO    MainThread:709772 [wandb_init.py:init():423] backend started and connected
2021-05-19 22:58:09,625 INFO    MainThread:709772 [wandb_init.py:init():465] updated telemetry
2021-05-19 22:58:09,625 INFO    MainThread:709772 [wandb_init.py:init():484] communicating current version
2021-05-19 22:58:10,760 INFO    MainThread:709772 [wandb_init.py:init():489] got version response 
2021-05-19 22:58:10,760 INFO    MainThread:709772 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-19 22:58:11,220 INFO    MainThread:709772 [wandb_init.py:init():522] starting run threads in backend
2021-05-19 22:58:16,227 INFO    MainThread:709772 [wandb_run.py:_console_start():1565] atexit reg
2021-05-19 22:58:16,228 INFO    MainThread:709772 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-19 22:58:16,229 INFO    MainThread:709772 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-19 22:58:16,229 INFO    MainThread:709772 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-19 22:58:16,229 INFO    MainThread:709772 [wandb_init.py:init():546] run started, returning control to user process
2021-05-19 22:58:25,983 INFO    MainThread:709772 [wandb_run.py:finish():1208] finishing run ranuga-d/Heart-Disease-UCI/cqulxkfg
